playground/css-lightningcss/nested/nested.css

```
@@ -0,0 +1,3 @@
+.nested-css-relative-asset {
+  background-image: url('../test.avif');
```

Fix looks good to me, but this test doesn't have any visual feedback, so I was a little confused.
I think doing it similar to `.external` would be nicer?


```suggestion
  background-image: url('../ok.png');
  width: 50px;
  height: 50px;
```